honeybee_3dm/config.py

```diff
@@ -177,35 +138,40 @@ def check_rad(config):
         KeyError: If the config file does not have a key named "radiance_material" in
             sources and a radiance material is requested in one of the layers in the
             config file.
-        ValueError: If any of the radiance identifiers mentioned in the config file 
+        ValueError: If any of the radiance identifiers mentioned in the config file
             are not found in the radiance materials file or do not match the identifiers
             in the radiance material file.
 
     Returns:
         Boolean value of True if no error is raised.
     """
 
-    # Check-04: Check if radiance modifiers can be imported
+    # Check if radiance modifiers can be imported
+    radiance = False
     for layer in config['layers']:
         if 'radiance_material' in config['layers'][layer]:
+            radiance = True
             break
+    if radiance:
```

Comment:
Python provides a `for/else` options.
```suggestion
    for layer in config['layers']:
        if 'radiance_material' in config['layers'][layer]:
            break
    else:
```

Reply:
for/else construct will be useful to do something once the loop has finished. Here, we don't want the loop to finish.

Reply:
Let me make sure that I understand the code here. Is it supposed to be for each layer or all the layers? Because the code that you have right now breaks the loop as soon a sit finds a radiance_material key in one of the layers similar to what I have suggested. Or am I missing something here?

Reply:
What you understood is correct and that's what I am trying to do. This means I am missing something from what you recommended. I need to look again at for/else.

Reply:
```
animals = ['cat', 'dog', 'human', 'horse']
for animal in animals:
    print(animal)
    if animal == 'human':
        break
else:
    print(animals)
```
In the code above, `animals` will not be printed. which is what we want.